### PR TITLE
Support task placement strategies other than random

### PR DESCRIPTION
**Pull request by kevinoriordan**

### CODE REVIEW

File: app/common/SundialGlobal.scala [hunk 1]
```diff
@@ -50,6 +51,19 @@ object SundialGlobal extends GlobalSettings {
     stack.getOutputs.asScala.find(_.getOutputKey == "WebAddress").get.getOutputValue
   }
 
+  lazy val taskPlacementStrategy = {
```
Comment (Plippe) [inline app/common/SundialGlobal.scala]:
```
val conf = play.Play.application.configuration

val taskPlacementString = conf.getString("ecs.defaultTaskPlacement", "random").toLowerCase
...
```

Comment (Plippe) [inline app/common/SundialGlobal.scala]: I think the configuration can also be injected

File: conf/application.conf [hunk 2]
```diff
@@ -14,6 +14,14 @@ db.default.password=password
 notifications.from="Sundial <sundial@yourdomain.com>"
 notifications.mode=email
 
+companion.tag=
```
Comment (Plippe) [inline conf/application.conf]: No value ??? is this ok

Comment (kevinoriordan) [inline conf/application.conf]: Yeah it's open sourced so someone has to deploy their own companion container to their own ECR in order to set stuff up.

**Pull request merged**